react/features/base/testing/components/TestConnectionInfo.js

```diff
@@ -184,6 +192,9 @@ class TestConnectionInfo extends Component<Props, State> {
                 <TestHint
                     id = 'org.jitsi.meet.conference.joinedState'
                     value = { this.props._conferenceJoinedState } />
+                <TestHint
+                    id = 'org.jitsi.meet.conference.grantModeratorEnabled'
+                    value = { this.props._grantModeratorEnabled } />
```

Comment:
Just say true here. Any version of this frontend will run with an up to date enough lib-jitsi-meet, so you can just say true here.

Reply:
Okai

Reply:
done!